lessons/r/dplyrmagrittr/lesson.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: dplyr and magrittr
+visible: true
+tags:
+  - r
+  - intermediate
+---
+
+ - **Authors**: Ahmed Hasan
+ - **Research field**: Genomics
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/r/dplyrmagrittr>
+
+## Topics Covered:
+
+- 'tidy' data
+- `dplyr` verbs:
+  - `select`
+  - `filter`
+  - `arrange`
+  - `mutate`
+  - `summarise`
+- `magrittr`
+  - the pipe (`%>%`)
+  - tee operator (`%T>%`)
+  - compound assignment (`%<>%`)
+
+### What You Will Need
+
+- R 3.3.1+ with `dplyr` and `magrittr` installed
+  - To install both packages, run `install.packages(c('dplyr', 'magrittr'))` in the command line
+
+If you'd like to follow along in Jupyter Notebook, do the following:
```

Comment:
I would recommend sticking to the respective IDEs for each language. Reduces the cognitive load everyone has for learning/trying out something new.